spec/system/admin_user_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+RSpec.describe 'ユーザ登録・ログイン・ログアウト機能', type: :system do
+  before do
+    FactoryBot.create(:user)
+    FactoryBot.create(:admin_user)
```

Comment:
こちら設定ファイルに追記がされていれば、省略が可能となります。
```suggestion
    create(:user)
    create(:admin_user)
```

Reply:
a419be6  にて修正しました。